src/shelly_common.hpp

```diff
@@ -51,6 +52,8 @@
 #define SHELLY_HAP_IID_BASE_OCCUPANCY_SENSOR 0x800
 #define SHELLY_HAP_IID_BASE_CONTACT_SENSOR 0x900
 #define SHELLY_HAP_IID_STEP_SENSOR 0x10
+#define SHELLY_HAP_IID_BASE_VALVE 0xa00
+#define SHELLY_HAP_IID_STEP_VALVE 4
```

Comment:
valve can have more characteristics. let's leave some space, just in case we ever need it. set step to 0x10